app/api/orders.py

```diff
@@ -115,6 +115,9 @@ def before_create_object(self, data, view_kwargs):
             data['is_billing_enabled'] = True
 
         free_ticket_quantity = 0
+        if data.get('status') != 'initializing':
+            raise ConflictException({'pointer': '/data/attributes/status'},
+                                    "status cannot be {} while creating an order".format(data.get('status')))
```

Comment:
@iamareebjamal Not sure if we need this check again as we are already making sure that status of the order is `initializing` in `before_post` function above in line 104.